Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -826,6 +826,9 @@ static void changeContentOffsetBoundedInValidRange(UIScrollView *scrollView, Web
 - (void)_didCommitLayerTreeDuringAnimatedResize:(const WebKit::RemoteLayerTreeTransaction&)layerTreeTransaction
 {
     auto updateID = layerTreeTransaction.dynamicViewportSizeUpdateID();
+
+    WTFLogAlways("THORTON %f: _didCommitLayerTreeDuringAnimatedResize update %llu want %llu", MonotonicTime::now().secondsSinceEpoch().value(), updateID.value_or(0), _currentDynamicViewportSizeUpdateID);
```

Comment:
Not sure you want THORTON in here.

Reply:
Or was that just to debug the timeout on EWS?

Reply:
(From the commit message, this is temporary logging to try and suss out the timeout on EWS)

Reply:
I do!!

Reply:
(two commits, one is the already-reviewed by wenson patch, one is "why in the world is it failing constantly on EWS")